operators/pkg/controller/elasticsearch/volume/volume.go

```diff
@@ -36,6 +36,9 @@ const (
 
 	ElasticsearchDataVolumeName = "elasticsearch-data"
 	ElasticsearchLogsVolumeName = "elasticsearch-logs"
+
+	ScriptsVolumeName      = "elastic-internal-scripts"
+	ScriptsVolumeMountPath = "/mnt/elastic/scripts"
```

Comment:
With the new naming this should be `/mnt/elastic-internal/scripts`.

Reply:
Thanks, fixed.